docs/develop/topics/i18n.md

```diff
@@ -71,24 +71,137 @@ Unicode - UTF8
 After strings have been marked for translation, it is now time to extract them. Instructions on how to do so can be found in the respective package in the file located at `.tx/config`. Depending on the packag, only Python or JavaScript may be available for translation.
 
 ### Python
+
+#### Configuration
+
+- In the root folder of your package update the `.tx/config` file:
+
+```
+[main]
+host = https://www.transifex.com
+
+# python conf
+[<parent_namespace_of_your_transifex_projext>.<package_name>-messages]
+file_filter = <package_name>/translations/<lang>/LC_MESSAGES/messages.po
+source_file = <package_name>/translations/messages.pot
+source_lang = en
+type = PO
+
+```
+
+- also in the root folder add `babel.ini` if not there:
+
+```
+# Extraction from Python source files
+[python: **.py]
+encoding = utf-8
+
+# Extraction from Jinja2 templates
+[jinja2: **/templates/**.html]
+encoding = utf-8
+extensions = jinja2.ext.autoescape, jinja2.ext.with_
+```
+
```

Comment:
- also in the root folder add these configuration line in `setup.cfg` if not there:
```
[compile_catalog]
directory = <package_name>/translations/
use-fuzzy = True
```
- remember to add this line for `MANIFEST.in`
```
recursive-include invenio_communities *.po *.pot *.mo
```
- also the entypoint in `setup.py`
```bash
    entry_points={
    ....
            'invenio_i18n.translations': [
            'messages = <package_name>',
        ],
    },
    ```